src/ae/oracle.js

```diff
@@ -177,7 +177,8 @@ async function registerOracle (queryFormat, responseFormat, options = {}) {
  * @return {Promise<Object>} Query object
  */
 async function postQueryToOracle (oracleId, query, options = {}) {
-  const opt = R.merge(this.Ae.defaults, options)
+  const oracle = await this.api.getOracleByPubkey(oracleId)
```

Comment:
```suggestion
  const queryFee = options.queryFee  || (await this.api.getOracleByPubkey(oracleId)).queryFee
```
then it won't fetch oracle if `queryFee` is provided